### PR TITLE
Store keys on read record. Send only stored keys values on update record

### PR DESCRIPTION
**Pull request by vvs-code**
Store keys on extracting record. Use stored keys on updating record

### CODE REVIEW

File: src/serializer.js [hunk 1]
```diff
@@ -16,7 +16,28 @@ var Serializer = DS.RESTSerializer.extend({
...
+  },
+  pickQueriedFields: function(data, record) {
+    var propsKeys = Object.keys(record.get('data')),
+      retData = {};
+
+    // skip pick-logic for CREATE requests
+    if(!propsKeys.length) {
+      retData = data;
+    } else {
+      propsKeys = Object.keys(record.get('data'));
```
Comment (andrewfan) [inline src/serializer.js]: propsKeys was already defined
https://github.com/collectrium/ember-data-socketio-adapter/pull/37/files#diff-4568d5ad520ddb6de4962d49a9de1047R25

File: src/serializer.js [hunk 2]
```diff
@@ -16,7 +16,28 @@ var Serializer = DS.RESTSerializer.extend({
   },
   extractDeleteRecords: function(store, type, payload) {
     return this.extractArray(store, type, payload);
+  },
+  serialize: function(snapshot, options) {
+    var hash = this._super(snapshot, options);
+    return this.pickQueriedFields(hash, snapshot, options);
+  },
+  pickQueriedFields: function(data, record) {
```
Comment (andrewfan) [inline src/serializer.js]: method accepts 2 args but you pass 3 https://github.com/collectrium/ember-data-socketio-adapter/pull/37/files#diff-4568d5ad520ddb6de4962d49a9de1047R22

**Comment by andrewfan** [comment]
plz rebase in 1 commit, and manually change release version to 0.1.37

File: src/serializer.js (hunk 2 above)
Comment (vvs-code) [inline src/serializer.js]: Fixed

File: src/serializer.js (hunk 1 above)
Comment (vvs-code) [inline src/serializer.js]: Thx

**Pull request merged**